svelte/src/routes/v2/scripts/draw/index.ts

```diff
@@ -74,11 +75,11 @@ function createInnerSimulation(
 export function draw(
 	svgElement: any,
 	graphData: any,
+	config: ConfigInterface,
```

Comment:
I am trying not to put config as the parameter here. as it blurred the line between drawing and filter. I would put the config inside the graphData (adding color for example)

Reply:
Agree, and this is kinda ugly. But updating drawdata whenever the config is updated is also ugly and I dont think its worth the hassle quite yet.